assets/js/script.js

```diff
@@ -32,12 +32,13 @@ $(document).ready(function() {
             //for J i=number
             //strIngredients + 
             for ( var i = 1; i < 21; i++) {   
+                //$("#temp").empty();
             result[`strIngredients${i}`]//creat a new li
             result[`strMeasure${i}`]
             //append to a single list item
             var listitem = `<ol class= "ingredients" ${result[`strIngredients${i}`]}
                                                     
-            </li>`;
+            </ol>`;
                
             $('.accordion-content').append(listitem);
```

Comment:
This will append to ALL `accordion-content` classes, what you're looking for is probably more along the lines of `$(this).append(listitem)` (I assume `$(this)` is the accordion list user clicked)

Reply:
You can test this by doing `console.log($('.accordion-content')` and seeing how many results are returned